package.json

```diff
@@ -91,14 +106,14 @@
     "tsup": "latest",
     "typescript": "^5.4.5",
     "undici-types": "^6.16.0",
-    "@solidjs/start": "^1.0.6",
-    "@sveltejs/kit": "^2.5.18",
-    "hono": "^4.5.8",
-    "h3": "^1.12.0",
     "vitest": "latest"
   },
   "dependencies": {
+    "@solidjs/start": "^1.0.6",
```

Comment:
We should not put these into dependencies, or devDependencies. They should be in optionalDependencies instead